src/nxdoc/nuxeo-server/installation/docker-image/build-a-custom-docker-image.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Build a Custom Docker Image
+review:
+  date: '2021-04-23'
+  status: ok
+labels:
+  - multiexcerpt
+  - multiexcerpt-include
+toc: true
+description: Discover how to build a custom Docker image.
+tree_item_index: 100
+---
+
+To build an application from Nuxeo, we strongly recommend to customize the Nuxeo Docker Image by building a Docker image from the Nuxeo one. Let's see how you can do this.
+
+## Build a Custom Docker Image From the Nuxeo One
+
+You can simply write a [Dockerfile](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) using the Nuxeo image as [parent image](https://docs.docker.com/glossary/#parent_image).
+A good practice is to use a [build argument](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact) before the `FROM` instruction to easily set the version of the Nuxeo parent image, as in the example below:
+
+```Dockerfile
+ARG NUXEO_VERSION=latest
+
+FROM docker.packages.nuxeo.com/nuxeo/nuxeo:${NUXEO_VERSION}
```

Comment:
I don't see the 10.10 Docker image in packages.nuxeo.com

Reply:
Nope, the hotfixed images live in docker-private.packages.nuxeo.com, see https://packages.nuxeo.com/#browse/search/docker=attributes.docker.imageName%3Dnuxeo%2Fnuxeo%20AND%20attributes.docker.imageTag%3D10.10*.

Reply:
This won't work.
Again, the LTS-2019 image lives in DockerHub, see https://hub.docker.com/_/nuxeo, so:
```
FROM nuxeo:${NUXEO_VERSION}
```